robot_number: 2
branch_number: 2
frontier_number: 4
robot1_x: 7.005
robot1_y: 8.26
robot1_yaw: 86.4
robot2_x: 23.817
robot2_y: 0
robot2_yaw: -180
branch1_x: 8.406
branch1_y: 12.18
branch2_x: 5.604
branch2_y: 12.18
frontier1_x: 34.091
frontier1_y: 0
frontier2_x: 42.03
frontier2_y: 10.78
frontier3_x: 5.137
frontier3_y: 12.18
frontier4_x: 8.873
frontier4_y: 11.62
